src/main/java/org/example/NumberRangeAnswer.java

```diff
@@ -0,0 +1,15 @@
+package org.example;
+
+import jakarta.persistence.Entity;
+
+@Entity
+public class NumberRangeAnswer extends AbstractAnswer implements Answer{
+    private String answer;
```

Comment:
Should we have it as a string or int?

Reply:
we could have two separate vars as the min and max of the range, or even an array of size 2.  but otherwise it would have to be a string since you cant express a range with primitive types

Reply:
oh okay, yea if we just expect a single int value in a range then it should be an int as far as i can tell